src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -9,7 +9,7 @@
     <InvariantGlobalization>true</InvariantGlobalization>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <TieredPGO>true</TieredPGO>
-    <PublishReadyToRun>true</PublishReadyToRun>
+    <PublishReadyToRun>false</PublishReadyToRun>
```

Comment:
Maybe it's better just to delete the line.

```suggestion
```

Reply:
sure